lib/ansible/modules/cloud/ovirt/ovirt_affinity_label_info.py

```diff
@@ -26,12 +26,14 @@
 
 DOCUMENTATION = '''
 ---
-module: ovirt_affinity_label_facts
-short_description: Retrieve facts about one or more oVirt/RHV affinity labels
+module: ovirt_affinity_label_info
+short_description: Retrieve information about one or more oVirt/RHV affinity labels
 author: "Ondra Machacek (@machacekondra)"
 version_added: "2.3"
 description:
-    - "Retrieve facts about one or more oVirt/RHV affinity labels."
+    - "Retrieve information about one or more oVirt/RHV affinity labels."
+    - This module was called C(ovirt_affinity_label_facts) before Ansible 2.9, returning C(ansible_facts).
+      Note that the M(ovirt_affinity_label_info) module no longer returns C(ansible_facts)!
 notes:
     - "This module creates a new top-level C(ovirt_affinity_labels) fact, which
```

Comment:
`ovirt_host_storage_info` and `ovirt_event_info` don't have this note, all other modules do have, though. I've updated the note in all other modules to mention return variable (instead of top-level fact), and that it needs to be registered so it can be used.